model.py

```diff
@@ -2487,19 +2619,11 @@ def parse_image_meta(meta):
     active_class_ids = meta[:, 8:]
     return image_id, image_shape, window, active_class_ids
 
-
-def parse_image_meta_graph(meta):
```

Comment:
this function is used in training phase,  should not be deleted

Reply:
Definitely, I did not plan on committing this part. If you know the solution to the current problem, I can clean this up asap. You can view the current issue in the pull request's last comment.

Reply:
Just checked it out! I got your point that the parse_image_meta_graph is being used in the training phase, my point was to remove the redundant function of parse_image_meta.  We'll do the clean up once our current errors are resolved. @LandyGuo Thanks for pointing it out 👍

Reply:
Your work is very impressive !  I have made some changes base on your implementation, and now everything seems fine. 

please refer to: https://github.com/LandyGuo/Mask_RCNN/blob/master/debug_detection_graph.py

Reply:
Awesome job @LandyGuo 👍 Do you mind submitting a pull request so we can merge into the original pull request and get the reviews/testing underway?

Reply:
@LandyGuo Thanks for this ! I tried / tested your implementation on my side and it seems to work fine (final detections seem identical). No problem when exporting and then evaluating the frozen graph.

For consistency, I would just replace `scores_select_size = class_ids.shape[0]` by `scores_select_size = tf.shape(probs)[0]`

Reply:
@ericj974  Thanks for the review! Great to hear it works well end to end. @LandyGuo Thanks for the awesome contribution. @Cpruce  I'll consolidate all the changes and send a PR to tonight so that we can merge this as soon as possible.

Reply:
Sounds good. Can you merge the incremental changes into mine? I was going to do it if Landy didn't want to but it seems like Landy fell off the face of the Earth... he never responded to me reaching out about him claiming credit for his contribution.